code/index.html

```diff
@@ -5,8 +5,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Project 4: Pizzabot</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <h1>This is your Pizza Bot 🤖🍕</h1>
+
+    <button id="btn" onclick="startOrder()">Order Pizza</button>
+
+    <div class="chat-box gray-color" id="welcome-box"> </div> 
+    <div class="chat-box gray-color" id="ourPizzaMenu"> </div>
+    <div class="chat-box blue-color" id="nameOfPizza"> </div>
+    <div class="chat-box gray-color" id="pizzaReply"> </div>
+    <div class="chat-box blue-color" id="numberOfPizzas"></div>
+    <div class="chat-box gray-color" id="confirmation-box"></div>
```

Comment:
If you have some time, check out `appendChild`
https://www.w3schools.com/jsref/met_node_appendchild.asp

That could be a way to create these rows "on-the-fly"